### PR TITLE
4 define models

### PR DESCRIPTION
**Pull request by miggitymack**
#4 All of the models are not set up. Users, tasks, and bounties are temporarily set up with associations to implement first stages of GraphQL server.

### CODE REVIEW

File: db/models/task.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+const { STRING } = require('sequelize');
+
+module.exports = db =>
+  db.define('tasks', {
+    description: STRING,
+    categories: STRING,
+    creator: STRING,
+    assignee: STRING
```
Comment (dffattal) [inline db/models/task.js]: Creator and Assignee need to come from two different associations to User:

```Task.belongsTo(User, {as: 'creator'})```
```Task.belongsTo(User, {as: 'assignee'})```

This adds a ```creatorId``` and ```assigneeId``` field to Task, and allows for the methods task.setCreator() and task.setAssignee()

Comment (miggitymack) [reply db/models/task.js]: Thanks for the catch

File: db/models/task.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+module.exports = db =>
+  db.define('tasks', {
+    description: STRING,
+    categories: STRING
+  });
+
+module.exports.associations = (Bounty, { Task, BountyTask }) => {
+  Task.belongsToMany(Bounty, { through: BountyTask });
+  Task.belongsTo(User, { as: 'creator' });
+  Task.belongsTo(User, { as: 'assignee' });
```
Comment (dffattal) [inline db/models/task.js:14]: I'm not actually sure how to set these associations now, will have to keep experimenting.

**Review by dffattal — APPROVED** [review]
Initial model mockup looks good, more models to be added later.

**Pull request merged**